unison-src/transcripts/add-test-watch-roundtrip.md

```diff
@@ -0,0 +1,15 @@
+```ucm:hide
+.> builtins.mergeio
```

Comment:
Off topic: why do both `builtins.merge` and `builtins.mergeio` exist, and is there a reason to prefer `mergeio` here?

Reply:
Good question. `builtins.merge` just includes "builtins" (from the point of view of the runtime). So that's primitive types like `Int`, `Nat`, primitive terms like `Nat.+`.

`builtins.mergeio`, includes all the "builtins" but also all the user-defined types that UCM commands rely on. So `Doc`, `Test.Result`, `Author`, etc.

`IO` used to be a user-defined ability, hence the name "builtins.mergeio", and the rest of the user-defined definitions were lumped in there as well for convenience. Today, `IO` is a primitive ability, so you actually get it from `builtins.merge`, and not from `builtins.mergeio` 😓

Reply:
`builtins.merge` takes less time to execute, so I prefer that when I can get away with it, since 2-3 seconds per transcript (or whatever it takes to run the parser/typechecker like `builtins.mergeio` must do) gets multiplied by 100 transcripts or whatever.

Kind of like:
`builtins.merge` fastest
`builtins.mergeio` fast-ish
`project.create` clone `@unison/base` 😑

Reply:
I was mistaken, `Test.Result` is in the smaller `builtins.merge`.